oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnectorConfig.java

```diff
@@ -86,6 +87,11 @@ public String getConnectionString() {
   @Nullable
   private String transactionIsolationLevel;
 
+  @Name(OracleConstants.USE_SSL)
+  @Description("Turns on SSL encryption. Connection will fail if SSL is not available")
+  @Nullable
+  public String useSSL;
```

Comment:
Why not `boolean`?

When we add something in plugin config, if we don't add a widget json for this. It will show up as a textbox in CDAP UI as well. 

If we don't want it to be visible in CDAP UI, please add a config in widgets json which is hidden.

Reply:
``` Why not boolean? ```
There was no specific reason, change it to Boolean.

```
When we add something in plugin config, if we don't add a widget json for this. It will show up as a textbox in CDAP UI as well.

If we don't want it to be visible in CDAP UI, please add a config in widgets json which is hidden.
```

got it, added it in the config of Batch Source and Batch Sink as only these are using OracleConnectorConfig. [8d62fb9cc4a75433d844f5e5285e0dcd690ea88b]

Reply:
There will also be a widget json file for `OracleConnector` which also need to be modified the same way.

Reply:
Updated [b68ec933ff27ff2a1ade2c578727113604803367]